test/testFeatures.m

```diff
@@ -19,16 +22,30 @@ function delete_html_file(testCase, data_path)
         %Checking the viewer runs on SPM-nidm input.
         function checkViewerRunsSPM(testCase)
             data_path = fullfile(fileparts(mfilename('fullpath')), '..', 'Data', 'ex_spm_default');
+            if(~exist(data_path, 'dir'))
+                mkdir(data_path)
+                websave(fullfile(data_path, 'tmp.zip'), 'http://neurovault.org/collections/1692/ex_spm_default.nidm.zip');
+                unzip(fullfile(data_path, 'tmp.zip'), fullfile(data_path, '.'));
+                
+                %Save the download link in the json.
+                json = spm_jsonread(fullfile(fileparts(mfilename('fullpath')), '..', 'Data', 'jsons', 'ex_spm_default.json'));
+                [designMatrix, dmLocation] = searchforType('nidm_DesignMatrix', json.x_graph);  
+                designMatrix{1}.prov_atLocation.x_value = 'http://neurovault.org/collections/1692/ex_spm_default.nidm/DesignMatrix.csv';
```

Comment:
My message was indeed not very clear... I guess my question was: why are you overwriting the location of the design matrix in:
```
designMatrix{1}.prov_atLocation.x_value = 'http://neurovault.org/collections/1692/ex_spm_default.nidm/DesignMatrix.csv'
```
while the viewer could directly use the design matrix csv file that is stored in the .nidm.zip?